schemas/lockfile.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/denoland/vscode_deno/blob/main/schemas/lockfile.schema.json",
+  "title": "Deno Lock file",
+  "description": "An import map which is used to remap imports when modules are loaded.\n\nSee https://deno.land/manual@v1.6.0/linking_to_external_code/import_maps in the Deno documentation for more information.",
+  "type": "object",
+  "required": [
+    "version",
+    "remote"
+  ],
```

Comment:
This file needs to be updated

Reply:
Fixed description of both the root and the sub-property, version